src/Features/Core/Portable/PdbSourceDocument/PdbSourceDocumentMetadataAsSourceFileProvider.cs

```diff
@@ -171,20 +173,23 @@ internal sealed class PdbSourceDocumentMetadataAsSourceFileProvider : IMetadataA
                 }
             }
 
+            Debug.Assert(!sourceDocuments.IsEmpty);
```

Comment:
This was already checked a few lines earlier, why the additional assert?

Reply:
Removed.